src/components/stories/index.js

```diff
@@ -11,9 +13,54 @@ const Stories = () => {
   const privateStories = stories.filter((s) => s.visibility === "PRIVATE");
   const publicStories = stories.filter((s) => s.visibility === "PUBLIC");
 
+  const [image, setImage] = useState(null);
+  const [story, setStory] = useState(null);
+
+  const create = () => {
+    storyService.createStory("me", story, image);
```

Comment:
If there is no image selected, then we shouldn't allow story creation. We can either add a validation here to not create the story if `image == null` or we can disable the `Create Story` button if `image == null`

Reply:
It is disabled when there is no image selected